frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -234,6 +234,19 @@ async function emailConfirmation(activityId: string) {
   };
   await submissionService.emailConfirmation(emailData);
 }
+
+async function checkValidility(event: Event) {
```

Comment:
Function name is unclear. What are we checking the validity of?

Reply:
added ActivityId for clarity